open_discussions/tests/app_tests.py

```diff
@@ -0,0 +1,13 @@
+"""Application level tests"""
```

Comment:
A couple things: 
-  `open_discussions/app_tests.py` still exists, so this seems redundant
- Adding this `tests/` directory is confusing. The pattern in this codebase and all of our other codebases is have test files as siblings to the files they test, and to avoid grouping them into a `tests/` directory. I think it would make more sense to get rid of this directory and to rename `render_bundle_test.py` to `templatetags_test.py` or something like that